packages/cfd/src/Containers/cfd-dbvi-onboarding.tsx

```diff
@@ -90,7 +100,6 @@ const CFDDbviOnboarding = ({
         }
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [is_cfd_verification_modal_visible]);
-
```

Comment:
why did you delete here the empty line? :)

Reply:
Honestly nt sure :)